frontend/src/pages/appointmentsettings/AppointmentStepsWindow.tsx

```diff
@@ -48,7 +137,31 @@ export default function AppointmentStepsWindow(
                         <React.Fragment>
 
                             <Typography component={'span'} sx={{mt: 2, mb: 1}}>
-                                <AppointmentStepsPages activepage={activeStep}/>
+                                <AppointmentStepsPages
+                                    activepage={activeStep}
+                                    //AppointOne
+                                    tastyDateName={tastyDateName}
+                                    setTastyDateName={setTastyDateName}
+                                    location={location} setLocation={setLocation}
+                                    notes={notes} setNotes={setNotes}
+                                    chosenDisplayName={chosenDisplayName}
+                                    setChosenDisplayName={setChosenDisplayName}
+                                    //AppointTwo
+                                    date={date} setDate={setDate}
+                                    startTime={startTime} setStartTime={setStartTime}
+                                    endTime={endTime} setEndTime={setEndTime}
+                                    idPickedTime={idPickedTime} setIdPickedTime={setIdPickedTime}
+                                    dataDateTimes={dataDateTimes} setDataDateTimes={setDataDateTimes}
+                                    //AppointThree
+                                    category={category} setCategory={setCategory}
+                                    postcode={postcode} setPostcode={setPostcode}
+                                    city={city} setCity={setCity}
+                                    restaurantName={restaurantName} setRestaurantName={setRestaurantName}
+                                    rating={rating} setRating={setRating}
+                                    price={price} setPrice={setPrice}
+                                    idPickedRestaurant={idPickedRestaurant} setIdPickedRestaurant={setIdPickedRestaurant}
+                                    restaurantData={restaurantData} setRestaurantData={setRestaurantData}
```

Comment:
i would maybe comine the props to the custom hook object? 🤔

Reply:
Thanks again for your help!